core/cloudflow-spark-testkit/src/main/scala/cloudflow/spark/testkit/SparkStreamletTestkit.scala

```diff
@@ -212,32 +218,27 @@ final case class SparkStreamletTestkit(session: SparkSession, config: Config = C
       outletTaps: Seq[SparkOutletTap[_]],
       duration: Duration
   ): Unit = {
-    val ctx = new TestSparkStreamletContext("streamlet-under-test", session, Seq(inletTap), outletTaps, config)
-    doRun(ctx, sparkStreamlet, duration)
+    val ctx = new TestSparkStreamletContext(TestStreamletName, session, Seq(inletTap), outletTaps, config)
+    doRun(ctx, sparkStreamlet)
   }
 
   private[testkit] def doRun(
       ctx: TestSparkStreamletContext,
-      sparkStreamlet: SparkStreamlet,
-      duration: Duration
+      sparkStreamlet: SparkStreamlet
   ): Unit = {
-    val t0             = System.currentTimeMillis()
-    def elapsedMillis  = System.currentTimeMillis() - t0
+    val t0           = System.currentTimeMillis()
+    def lap          = System.currentTimeMillis() - t0
+    val queryMonitor = new QueryExecutionMonitor()
+    ctx.session.streams.addListener(queryMonitor)
+    println("============================================== execution start =============" + lap)
```

Comment:
I think these printlns should be removed, it is not very common to force print something during a test, especially if users can't turn it off. (End-users also use the testkit of course)

Reply:
(maybe they are there for some debugging / testing?)

Reply:
:-)
To quote myself from the PR description:
>Note:
>This Draft PR still contains plenty of heisen(de)bug output to observe how it behaves in the >CI machinery.

Reply:
yeah apologies I obviously can't read text, only code 🤦‍♂